src/ds/d400/d400-device.h

```diff
@@ -94,6 +94,7 @@ namespace librealsense
         friend class d400_depth_sensor;
 
         std::shared_ptr<hw_monitor> _hw_monitor;
```

Comment:
Member `hw_monitor_response` is not needed, create the shared object as you directly pass it into `_hw_monitor`.
Also, should be prefixed with `_`